src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -5737,19 +5807,24 @@ public override BoundNode VisitLocalFunctionStatement(BoundLocalFunctionStatemen
                     snapshotBuilder = null;
                 }
 
+                LocalFunctionSymbol updatedLocalFunction = node.Symbol.WithContainingSymbol(GetUpdatedContainingSymbol(node.Symbol));
```

Comment:
>LocalFunctionSymbol updatedLocalFunction = node.Symbol.WithContainingSymbol(GetUpdatedContainingSymbol(node.Symbol)); [](start = 16, length = 117)

Same comment about creating symbols for every pass.

Reply:
No longer being recreated every pass.

---
In reply to: [327923291](https://github.com/dotnet/roslyn/pull/38769#discussion_r327923291) [](ancestors = 327923291)